Reason: Ignore type-limits error
Upstream: No

This issue is fixed by upstream in https://github.com/taocpp/PEGTL/commit/a3a292c2ba66bf09b4adc4cee8fcdfb1b1386067,
but the patch does not apply to current codebase (version 2.8.3).

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0fe7e8e7..a3fedc20 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,6 +7,8 @@ set(PEGTL_VERSION "${CMAKE_MATCH_1}")
 
 project(pegtl VERSION ${PEGTL_VERSION} LANGUAGES CXX)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error=type-limits")
+
 if(${PROJECT_NAME}_FOUND)
   # Multiple versions of PEGTL can't co-exist
   if(NOT ${PROJECT_NAME}_VERSION STREQUAL ${PROJECT_VERSION})
